flight_path_viewer.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from json import loads
+from fastkml import kml, geometry
+import argparse
+from sys import stdin, stdout
+
+
+def read_log_file(f):
+    lines = []
+    for line in f:
+        timestamp, data = line.split(";", 1)
+        timestamp = datetime.strptime(timestamp, "%Y-%m-%d %H:%M:%S,%f")
+        data = loads(data)["result"]["quick"]
+        lines.append((timestamp, data))
+    lines.sort()
+    return lines
+
+
+def gen_doc(entries, step=10):
+    k = kml.KML()
+    d = kml.Document()
+    k.append(d)
+    path = kml.Folder()
+    all_coords = []
+    for i in range(0, len(entries), step):
+        t, data = entries[i]
+        p = kml.Placemark()
+        time = kml.TimeStamp()
+        time.timestamp = t
+        p.timeStamp = time.timestamp
+        pt = (data["lon"], data["lat"], data["altitude"])
+        all_coords.append(pt)
+        p.geometry = geometry.Geometry(
+            geometry=geometry.Point(*pt),
+            altitude_mode="relativeToGround",
+        )
+        path.append(p)
+    d.append(path)
+    line_path = kml.Placemark(name="UAV flight path")
+    line_path.geometry = geometry.Geometry(
+        geometry=geometry.LineString(
+            all_coords,
+        ),
+        altitude_mode="relativeToGround",
+    )
+    d.append(line_path)
+    return k
+
+
+def main():
+    a = argparse.ArgumentParser(
+        prog="UAV flight path generator",
```

Comment:
...why?

Reply:
Specifically, why this:
```
usage: UAV flight path generator [-h] in_file out_file
```

instead of this:
```
usage: flight_path_viewer.py [-h] in_file out_file
```